### PR TITLE
Added technologies to README file

### LINKED ISSUE #178: Add  technologies to README file
**I'm submitting a ...**
- [x] feature request

**Current behavior:**
There is no technology section in the Readme

**Expected behavior:**
Readme should have  technologies we are using in the repository and how it work.

**Steps to reproduce:**
- Edit the README file
- Add technology information just like [this](https://github.com/fossasia/meilix-generator#technologies-used).

### PR DESCRIPTION
**Pull request by alirizwi**
Fixes #178 

#### Checklist

- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia) and my PR follows them.
- [x] My branch is up-to-date with the Upstream `development` branch.
- [x] I have added necessary documentation (if appropriate)

#### Changes proposed in this pull request:

- Added technologies to README file

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -13,7 +13,7 @@ The goal of badgeyay is to provide a simple badge generator with the following f
    
    
 To get a better idea about the working of badgeyay ,you can check out the following:
-[![Alt Watch](https://img.youtube.com/vi/Gh8j_01LIoQ/hqdefault.jpg)](https://www.youtube.com/watch?v=Gh8j_01LIoQ)
+[![Alt Watch](https://user-images.githubusercontent.com/8705386/30831526-438f8c4c-a237-11e7-83fc-c12046f12e18.png)](https://www.youtube.com/watch?v=Gh8j_01LIoQ)
```
Comment (vaibhavsingh97) [inline README.md]: Please don't change anything which is not related to PR

File: README.md [hunk 2]
```diff
@@ -28,6 +28,17 @@ If you like to join developing,
 Specification
 -------------
 
+### Technologies Used
+
+Badgeyay uses a number of open source projects:
+
+* [Flask](http://flask.pocoo.org/) - Microframework powered by python
+* [Bootstrap](http://getbootstrap.com/) - Responsive frontend framework
```
Comment (vaibhavsingh97) [inline README.md]: We are using Bootstrap v3 so i think we should redirect to https://getbootstrap.com/docs/3.3/

File: README.md (hunk 1 above)
Comment (jajodiaraghav) [reply README.md]: I think we can let this stay. As there is an issue that wants the thumbnail changed. What do you think?

**Comment by alirizwi** [comment]
@vaibhavsingh97 , I have made the changes as you said. Please review it.
I have to revert a commit because, by mistake i have made a commit from different branch.

**Comment by vaibhavsingh97** [comment]
@alirizwi Can you please squash your commits into one

**Comment by vaibhavsingh97** [comment]
@alirizwi Can you please squash your commits into one 😄

**Comment by alirizwi** [comment]
@vaibhavsingh97 @gabru-md @meets2tarun I have squashed the commits. Please review. Thanks
